internal/cmd/run/cypress.go

```diff
@@ -135,40 +134,15 @@ func runCypress(cmd *cobra.Command, isCLIDriven bool) (int, error) {
 	go func() {
 		props := usage.Properties{}
 		props.SetFramework("cypress").SetFVersion(p.GetVersion()).SetFlags(cmd.Flags()).SetSauceConfig(p.GetSauceCfg()).
-			SetArtifacts(p.GetArtifactsCfg()).SetDocker(p.GetDocker()).SetNPM(p.GetNpm()).SetNumSuites(len(p.GetSuites())).SetJobs(captor.Default.TestResults).
+			SetArtifacts(p.GetArtifactsCfg()).SetNPM(p.GetNpm()).SetNumSuites(len(p.GetSuites())).SetJobs(captor.Default.TestResults).
 			SetSlack(p.GetNotifications().Slack).SetSharding(p.IsSharded()).SetLaunchOrder(p.GetSauceCfg().LaunchOrder)
 
 		tracker.Collect(cases.Title(language.English).String(cmds.FullName(cmd)), props)
 		_ = tracker.Close()
 	}()
 
 	cleanupArtifacts(p.GetArtifactsCfg())
-	dockerProject, sauceProject := cypress.SplitSuites(p)
-	if dockerProject.GetSuiteCount() != 0 {
-		exitCode, err := runCypressInDocker(dockerProject)
-		if err != nil || exitCode != 0 {
-			return exitCode, err
-		}
-	}
-	if sauceProject.GetSuiteCount() != 0 {
-		return runCypressInSauce(sauceProject, regio)
-	}
-
-	return 0, nil
-}
-
-func runCypressInDocker(p cypress.Project) (int, error) {
-	log.Info().Msg("Running Cypress in Docker")
-	printTestEnv("docker")
-
-	cd, err := docker.NewCypress(p, &testcompClient, &testcompClient, &restoClient, &restoClient, createReporters(p.GetReporter(), p.GetNotifications(), p.GetSauceCfg().Metadata, &testcompClient, &restoClient,
-		"cypress", "docker"))
-	if err != nil {
-		return 1, err
-	}
-
-	p.CleanPackages()
-	return cd.RunProject()
+	return runCypressInSauce(p, regio)
```

Comment:
`runCypressInSauce` can likely be renamed then, since there's no other way to run it.

Reply:
I directly removed the function, as it was not bringing that much value to have one.